### ISSUE TITLE
CAM: New Simulator's task panel truncates translations for `Job` 

### ISSUE THREAD

**Issue by FEA-eng** [FreeCAD/FreeCAD#16707]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

I've noticed that the translation for `Job:` is cut in the task panel of the New Simulator:

![job 1](https://github.com/user-attachments/assets/44035fab-8e13-4bf0-bb8f-15f97b077c69)

![job 2](https://github.com/user-attachments/assets/f721aea2-0d8b-45a2-9600-00e4f0114d4b)

It should be `Zadanie:`. The same problem occurs with other languages.

### Full version info

```shell
OS: Windows 10 build 19045
Word size of FreeCAD: 64-bit
Version: 1.1.0dev.38794 (Git)
Build type: Release
Branch: main
Hash: a47800d9ccd7d7a7cdb1779d56de03d3c64cd76e
Python 3.11.9, Qt 5.15.13, Coin 4.0.3, Vtk 9.2.6, OCC 7.7.2
Locale: Polish/Poland (pl_PL)
Stylesheet/Theme/QtStyle: unset/FreeCAD Classic/Qt default
```

### Subproject(s) affected?

CAM

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by luzpaz** [comment]
@kaktusus have you seen this before ?

**Comment by kaktusus** [comment]
It wasn't until Kuba showed it to me.

![obraz](https://github.com/user-attachments/assets/0e7c02ae-5332-4ba9-a0d8-3813da1c3988)

On my machine I do not see this error :no_good:
I checked on releases from 38794 to 38467.

**Comment by kaktusus** [comment]
The problem is with the new CAM Simulation.

![obraz](https://github.com/user-attachments/assets/37b4a7fc-1536-48cd-8421-6d84ceddf446)

**Comment by luzpaz** [comment]
@shaise ping

**Comment by shaise** [comment]
Sorry, did not see that I'll take a look.

**Comment by shaise** [comment]
OK. Added a fix to my latest PR #16665.

![image](https://github.com/user-attachments/assets/0369b08c-1555-49d6-8ca4-e8e2b2b62a15)

**Comment by FEA-eng** [comment]
> OK. Added a fix to my latest PR #16665.

Great, thanks.

**Comment by kaktusus** [comment]
And is it possible to add another fix?
So that the untranslated "High" gets a translation :question:

**Comment by shaise** [comment]
No need to fix. High, Medium and Low are all translateable

**Comment by FEA-eng** [comment]
Fixed by #16665.
